libs/ktem/ktem/index/file/graph/pipelines.py

```diff
@@ -116,6 +118,17 @@ def call_graphrag_index(self, input_path: str):
         print(result.stdout)
         command = command[:-1]
 
+        # copy customized GraphRAG config file if it exists
+        setting = dotenv.dotenv_values(".env")
```

Comment:
We already used 
```
from decouple import config
value = config("USE_CUSTOMIZED_GRAPHRAG_SETTING", default="value")
```

for this.

Reply:
Good to know that we already have this config. Yes, it is definitely better to use it in the UI than in .env. Actually, I also considered adding a config in gradio when I submitted it, but, I was not familiar with Gradio, so I didn't do it at that time... Thank you guys.